File: src/0168-Excel-Sheet-Column-Title/0168.py
class Solution:
    def convertToTitle(self, n):
        """
        :type n: int
        :rtype: str
        """
        res = ''
        while n:
            n -= 1
            res += chr(ord('A')+n%26)
            n //= 26
            
        return res[::-1]

File: src/0168-Excel-Sheet-Column-Title/0168.cpp
static int x = []() {std::ios::sync_with_stdio(false); cin.tie(0); return 0; }();
class Solution 
{
public:
    string convertToTitle(int n) 
    {
        string res = "";
        while (n)
        {
            res = (char)('A' + --n%26) + res;
            n /= 26;
        }
        return res;
    }
};